.travis.yml

```diff
@@ -1,4 +1,7 @@
 gemfile: kubernetes/Gemfile
 rvm:
   - 2.6
-script: sh -c 'cd kubernetes && bundle exec rspec'
+
+install: gem install rubocop
```

Comment:
This shouldn't be needed because its in the gem file.  but then you  have to run it the same way we do `rspec` `bundle exec rubocop`

This way the version that runs in CI is the same. (you are currently using the globally installed version which might change)